README.md

```diff
@@ -49,7 +49,27 @@ Contains the JSON schemas used to validate the sources in the integration tests.
+| name        |  Text |Optional              | An optional description of the data source, e.g to specify if the data source is an aggregate of multiple providers
+| static_reference |  List of Integers |Optional              | A list of the static sources that the real time source is associated with, represented by their MDB source IDs. |  
+|note|Text| Optional|A note to clarify complex use cases for consumers, for example when several static sources are represented in a realtime source.  |                                                                                                                
+| vehicle_positions |List of JSON| Optional | List of vehicle positions data associated with the static reference list. [This JSON follows the Realtime URL data structure described below](https://github.com/MobilityData/mobility-database-catalogs/edit/main/README.md#realtime-url-data-structure).
+| trip_updates |List of JSON|Optional | List of trip updates data associated with the static reference list. [This JSON follows the Realtime URL data structure described below](https://github.com/MobilityData/mobility-database-catalogs/edit/main/README.md#realtime-url-data-structure)
+|service_alerts| List of JSON | Optional | List of service alerts data associated with the static reference list. [This JSON follows the Realtime URL data structure described below](https://github.com/MobilityData/mobility-database-catalogs/edit/main/README.md#realtime-url-data-structure)
+
+### Realtime URL data structure
+
+|Field Name|Type |Presence|Definition|                                                                     
+|-|-|-|-|
+|direct_download_url |URL|Optional     | URL that automatically opens.
+|license_url  |URL| Optional     | The license information for the direct download URL.                                                                                                                   
+|authentication_type |Enum|Conditionally required | The **authentication_type** field defines the type of authentication required to access the URL. When a direct download URL is provided, the authentication type is required. Valid values for this field are: <ul> <li>**0** or **(empty)** - No authentication required.</li><li>**1** - Ad-hoc authentication required, visit URL in `authentication_info_url` for more information. </li><li>**2** - The authentication requires an API key, which should be passed as value of the parameter `api_key_parameter_name` in the URL. Please visit URL in `authentication_info_url` for more information.</li></li>**3**: The authentication requires an HTTP header, which should be passed as the value of the header api_key_parameter_name in the HTTP request.</li><li>**4**: A placeholder text value is provided within the URL, which is surrounded by handlebars e.g {API_KEY}. Consumers should replace this value</li></ul>|
```

Comment:
If we want to encourage industry-standard authentication methods that are preferred by consumers, I'd suggest moving "ad-hoc" to the last value of `4`. I think this sends a message of "this is the last resort" option, and not your first choice if you're a producer implementing authentication.

Reply:
Current option 4 "Placeholder in URL" definition needs to change a little.

I had originally expected that `api_key_parameter_name` would hold the placeholder text value, in which case the definition would be:

><li>**4**: A placeholder text value is provided within the URL, as defined by `api_key_parameter_name`. Consumers should replace this placeholder text value with the value of the API key.</li></ul>

...and `api_key_parameter_name` would be required for this case.

I suppose, though, if MobilityData controls all the listings you could enforce the use of `{API_KEY}` as the placeholder, in which case the definition would be:

><li>**4**: A placeholder text value of `{API_KEY}` is provided within the URL. Consumers should replace `{API_KEY}` with the value of the API key.</li></ul>

I think examples are really useful here too to make sure the exact logic is understood.

Reply:
I think using `{API_KEY}` would be easier here, so that it's uniform throughout the catalog. A few things to consider:

- The urls provided through the Google Form may be using various placeholder text values. Should we be replacing them with `{API_KEY}` or leave them as is? I'd vote to replace them with `{API_KEY}`, but any thoughts are welcome here. 
- For the GTFS Realtime sources added directly by a third party though a PR, we will be able to ask the placeholder text value to be `{API_KEY}`.

Reply:
>The urls provided through the Google Form may be using various placeholder text values. Should we be replacing them with {API_KEY} or leave them as is? I'd vote to replace them with {API_KEY}, but any thoughts are welcome here.

For this to be consistent for consumers, I think we'd need to replace them. Maybe include strict language and example on the Google Form to help people submitting get this right?